code/modules/projectiles/guns/energy/special.dm

```diff
@@ -122,6 +122,8 @@
 		overheat = 0
 		recent_reload = 0
 	..()
+/obj/item/weapon/gun/energy/kinetic_accelerator/emp_act(severity)
+	return

```

Comment:
This is a bandaid over an earlier fix to ebows and KAs breaking when EMP'd. They are lever-action, and EMP shouldn't affect them.